packages/token-metadata/src/ibc/channels.ts

```diff
@@ -59,24 +60,26 @@ export const canonicalChannelsToChainMapFromInjective = {
   'channel-102': { chainA: 'Injective', chainB: 'Migaloo' },
   'channel-104': { chainA: 'Injective', chainB: 'Terra2' },
   'channel-105': { chainA: 'Injective', chainB: 'Terra2' },
+  'channel-143': { chainA: 'Injective', chainB: 'Kava' },
 }
 
-export const canonicalChannelsToChainMapToInjective = {
-  'channel-220': { chainA: 'CosmosHub', chainB: 'Injective' },
-  'channel-122': { chainA: 'Osmosis', chainB: 'Injective' },
-  'channel-17': { chainA: 'Terra', chainB: 'Injective' },
-  'channel-41': { chainA: 'Persistence', chainB: 'Injective' },
-  'channel-12': { chainA: 'Chihuahua', chainB: 'Injective' },
-  'channel-23': { chainA: 'Secret Network', chainB: 'Injective' },
-  'channel-59': { chainA: 'Juno', chainB: 'Injective' },
-  'channel-6': { chainA: 'Stride', chainB: 'Injective' },
-  'channel-90': { chainA: 'Crescent', chainB: 'Injective' },
-  'channel-1': { chainA: 'Sommelier', chainB: 'Injective' },
-  'channel-8': { chainA: 'Canto', chainB: 'Injective' },
-  'channel-91': { chainA: 'Terra2', chainB: 'Injective' },
-  'channel-92': { chainA: 'Terra2', chainB: 'Injective' },
-  'channel-3': { chainA: 'Migaloo', chainB: 'Injective' },
-}
+export const canonicalChannelsToChainMapToInjective = [
```

Comment:
Why don't we also add the above map here? `chainA` will always be injective in that case so we will have one array of objects instead of two. We just have to be careful to reflect these changes where these constants are used.